frameworks/digital-outcomes-and-specialists/questions/services/bespokeSystemInformation.yml

```diff
@@ -1,3 +1,8 @@
 question: >
-  If you build bespoke systems, do you recognise, and can you ensure, that the data will continue to be available for future government use. 
+  If you build bespoke systems, do you recognise, and can you ensure, that the data will continue to be available for future government use.
 type: boolean
+depends:
+  - on: "lot"
```

Comment:
`"on"` is the part we need to quote because of yaml's boolean conversion.

(This was ignored by tests because of an error in questions schema, I'm adding a fix in a new commit)